### ISSUE TITLE
oldpromise is not defined

### ISSUE THREAD

**Issue by alexjfno1** [Codeception/CodeceptJS#373]
I have just updated to `0.4.14` and after running my tests again I have been getting the error `oldpromise is not defined`.

After doing some poking in the code I found that the variable `oldpromise` has been renamed to `oldPromise` in commit [https://github.com/Codeception/CodeceptJS/commit/0fc4713fd47b4b7ee7f56571f6150037341f8cc6#diff-b97cf4f6778f69a84e7a2446d1b4dbdfL3](https://github.com/Codeception/CodeceptJS/commit/0fc4713fd47b4b7ee7f56571f6150037341f8cc6#diff-b97cf4f6778f69a84e7a2446d1b4dbdfL3)

Further down in the restore function it hasn't been renamed. Hence getting the error explained above. [https://github.com/Codeception/CodeceptJS/blob/master/lib/recorder.js#L56](https://github.com/Codeception/CodeceptJS/blob/master/lib/recorder.js#L56)

**Comment by DavertMik** [comment]
yes, sorry, some automated refactoring tools could have helper with this. 
Unfortunately this very part is not unit tested well.

**Comment by DavertMik** [comment]
I don't have any ideas why this haven't been caught by this test:

```js
    it('can be started saving previous promise chain', () => {
      let order = '';
      recorder.add(() => order += 'a');
      recorder.add(() => {
        recorder.session.start();
        recorder.add(() => order += 'c');
        recorder.add(() => order += 'd');
      });
      recorder.add(() => recorder.session.restore());
      recorder.add(() => order += 'b');
      return recorder.promise()
        .then(() => assert.equal(order, 'acdb'));
    });
```

Strange...

**Comment by alexjfno1** [comment]
👍  Brilliant thanks. Yeah no idea why test didn't catch it?
